### PR TITLE
Enable translation and localization

### PR DESCRIPTION
**Pull request by kaichaosun**
https://github.com/substrate-developer-hub/docs/issues/78

### CODE REVIEW

File: docs/quickstart/installing-substrate.md [hunk 1]
```diff
@@ -28,4 +28,4 @@ This may take several minutes to finish. Once it is done, check that Substrate i
   ]
 }
 [/block]
-If you get that, then you're ready to proceed!
\ No newline at end of file
+If you get that, then you're ready to proceed!
```
Comment (JoshOrndorff) [inline docs/quickstart/installing-substrate.md]: What changed on this line?

File: crowdin.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        'cs': 'cs'
+        'da': 'da'
+        'de': 'de'
+        'el': 'el'
+        'es-ES': 'es-ES'
+        'fa': 'fa-IR'
+        'fi': 'fi'
+        'fr': 'fr'
+        'he': 'he'
+        'hu': 'hu'
```
Comment (JoshOrndorff) [inline crowdin.yaml]: Does having _all_ of these languages mean that users will actually see that many choices? If so, we should slim it down to just English, Chinese, and any language we actually have significant content in.

File: docs/quickstart/installing-substrate.md (hunk 1 above)
Comment (kaichaosun) [reply docs/quickstart/installing-substrate.md]: Add an empty line at the end of the file.

File: crowdin.yaml (hunk 2 above)
Comment (kaichaosun) [reply crowdin.yaml]: This file is generated when running script ``, the flag to control whether the user can see the lang in drop down is controlled [here](https://github.com/substrate-developer-hub/docs/pull/93/files#diff-8690ce0767c84cc3ba0f00c5d757cf9dR10). 
Currently, I only enabled the English and Chinese.

**Comment by kaichaosun** [comment]
Rebase on master branch.

**Comment by JoshOrndorff** [comment]
@kaichaosun I'm not sure whether any of the repository-moving or anything else I've fiddled with broke this PR. I think this is good work, and would like to see it merged. Please merge it yourself when you have it merging again. :+1:

**Comment by kaichaosun** [comment]
I have rebased on the source branch. I don't have the access to merge the PR, who should I ask to give me the access? @JoshOrndorff

**Comment by JoshOrndorff** [comment]
@kaichaosun I've merged this one for you and invited you as a collaborator on this repo. Didn't realize you couldn't merge.

**Comment by shawntabrizi** [comment]
I think this PR requires some follow up. There are some links which need some JS logic in order to append the correct language to the URL.

Like this:

```
    const langPart = `${language ? `${language}/` : ""}`;
    const docUrl = doc => `${baseUrl}${docsPart}${langPart}${doc}`;
    const pageUrl = page => baseUrl + (language ? `${language}/` : "") + page;
```

**Comment by kaichaosun** [comment]
I will follow this up.

**Comment by kaichaosun** [comment]
@shawntabrizi I have opened a PR to fix the links, https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/102
Seems no need to change following js code:
```
const langPart = `${language ? `${language}/` : ""}`;
const docUrl = doc => `${baseUrl}${docsPart}${langPart}${doc}`;
const pageUrl = page => baseUrl + (language ? `${language}/` : "") + page;
```
I'm open to change if you have other thoughts.

**Pull request merged**